amundsen_application/static/js/ducks/tableMetadata/api/helpers.ts

```diff
@@ -13,10 +13,22 @@ export function getTableQueryParams(key: string, index?: string, source?: string
 }
 
 /**
- * Parses the response for table metadata to create a TableMetadata object
+ * Generates the query string parameters needed for the request for the related dashboards to a table
  */
-export function getTableDataFromResponseData(responseData: API.TableDataAPI): TableMetadata {
-  return filterFromObj(responseData.tableData, ['owners', 'tags']) as TableMetadata;
+export function getRelatedDashboardSlug(key: string): string {
+  return encodeURIComponent(key);
+}
+
+/**
+ * Parses the response for table metadata and the related dashboard information to create a TableMetadata object
+ */
+export function getTableDataFromResponseData(responseData: API.TableDataAPI, relatedDashboardsData: API.RelatedDashboardDataAPI ): TableMetadata {
+  const mergedTableData = {
+    ...filterFromObj(responseData.tableData, ['owners', 'tags']),
+    ...relatedDashboardsData
```

Comment:
Even though `RelatedDashboardDataAPI` is typed as  `{ dashboards: DashboardResource[]; }`, I think extra information like `msg` will still exist on that object and then get passed around and eventually set on the the returned object. Could you double check whether or not `mergedTableData` end up with only `dashboards` and not `msg`?

Reply:
I will double check!

Reply:
You are right! There is a msg and status_code that shouldn't be there, I will remove those.

Thanks!